recipes/cmake/3.x.x/test_package/conanfile.py

```diff
@@ -1,13 +1,17 @@
 import os
 from six import StringIO
-from conans import ConanFile, tools
+from conan import ConanFile
+from conan.tools.build import can_run
 
 
 class TestPackageConan(ConanFile):
     settings = "os", "arch"
```

Comment:
```suggestion
    settings = "os", "arch"
    test_type = "explicit"
    generators = "VirtualRunEnv"
```

Reply:
The `test_type` attribute is getting ignored in 2.x so it probably should be added to the 1.x test_package instead